source/INI/PeanutButter.INI/INIFile.cs

```diff
@@ -1058,6 +1066,7 @@ PersistStrategies persistStrategy
             var addSeparator = SectionSeparator is not null;
             var commentDelimiter = lineParser.CommentDelimiter;
             var sep = CommentIfNecessary(SectionSeparator, commentDelimiter);
+            var addEmptyLineAtTheBottom = AddEmptyLineAtTheBottom;

```

Comment:
no need to var off - just use the property inline at the end - I don't think INIFile is rated for thread safety anyway 😄

Reply:
Yeah, I agree. I was just trying to fit in the style.😁